pinot-query-planner/src/main/java/org/apache/calcite/rel/hint/PinotHintStrategyTable.java

```diff
@@ -27,8 +27,11 @@
  * Default hint strategy set for Pinot query.
  */
 public class PinotHintStrategyTable {
-  public static final String INTERNAL_AGG_INTERMEDIATE_STAGE = "aggIntermediateStage";
   public static final String INTERNAL_AGG_FINAL_STAGE = "aggFinalStage";
+  public static final String INTERNAL_AGG_INTERMEDIATE_STAGE = "aggIntermediateStage";
+  public static final String INTERNAL_AGG_LEAF_STAGE = "aggLeafStage";
```

Comment:
1. can we add javadocs for these? i think some definition is different from previous usage of these internal hints
2. please also include the internal hints to agg node function name mapping in the java doc and reference to those ( i believe it was called `MODE` in #10845)

Reply:
done